src/dc_receive_imf.rs

```diff
@@ -979,9 +979,14 @@ fn create_or_lookup_group(
     // XXX insert code in a different PR :)
 
     // execute group commands
-    if X_MrAddToGrp.is_some() || X_MrRemoveFromGrp.is_some() {
+    if X_MrAddToGrp.is_some() {
+        chat::reset_gossiped_timestamp(context, chat_id)?;
```

Comment:
So here we are _receiving_ a member added message which should already contain the gossip keys for this chat, including the added member. Why do we need to resent gossip ourselves?

Reply:
I have removed this line too, it is indeed not needed. Looks like it triggered gossip timestamp reset for all members in a group so they all send gossip with their next message, and it is not needed.

Reply:
> Why do we need to resent gossip ourselves?

sending to the same chat is not needed, but re-gossiping is also done in other chats.

Reply:
But in this case gossip timestamp is reset for this `chat_id` only, which is not required for sure.